app/views/planning_application/assessment_details/consultation_summary/_information.html.erb

```diff
@@ -6,3 +6,24 @@
     read_only: read_only
   }
 ) %>
+
+<% if @planning_application.consultees.where.not(response: nil).present? %>
```

Comment:
Can we add this as a scope on the consultee model i.e 

`scope :with_response, -> { where.not(response: nil) }`

guess you can even store this as an instance variable if calling multiple times